src/Components/Web/src/Virtualization/ItemsProviderResult.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information. 
+
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.Components.Web
```

Comment:
```suggestion
namespace Microsoft.AspNetCore.Components.Web.Virtualization
```

Same for all new types